### PR TITLE
First public release

### PR DESCRIPTION
**Pull request by matmas**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.7',
+        'Topic :: Software Development :: Libraries :: Python Modules',
+        'Framework :: Django',
+    ],
+    keywords='django rest framework client',
+    packages=find_packages(exclude=['tests']),
+    install_requires=[
+        'setuptools',
+        'Django',
```
Comment (tuomas2) [inline setup.py:32]: There is very light dependency on Django which could easily be removed. I would suggest doing this.

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    url='https://github.com/qvantel/django-rest-framework-client',
+    author="Martin Riesz",
+    author_email="ext-martin.riesz@qvantel.com",
+    classifiers=[
+        'Development Status :: 4 - Beta',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: BSD License',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.7',
```
Comment (tuomas2) [inline setup.py:24]: No Py3 support? I would consider py3 support very important at this point. Clock is ticking .. (https://pythonclock.org/) :P

**Review by tuomas2 — COMMENTED** [review]
Please add .travis.yml and set up running unit tests in travis

File: setup.py (hunk 1 above)
Comment (matmas) [reply setup.py:32]: Removing this dependency is a bit more involved. Let's do it iteratively.

File: setup.py (hunk 2 above)
Comment (matmas) [reply setup.py:24]: @axdel mentioned he has made it py3 compatible already and could make a separate pull request.

**Pull request merged**